general/development/policies/codingstyle/index.md

```diff
@@ -985,6 +985,113 @@ $string['overduehandlingautoabandon'] = 'Unfinished attempts are immediately dis
 
 Language strings should not contain or even rely on any leading or trailing whitespace. Such strings are not easy to be translated in the translation tool, AMOS.
 
+## Objects
+
+### Importing `stdClass`
+
+For namespaced classes, importing `stdClass` via the `use` operator or using its fully qualified name are both acceptable.
+
+<ValidExample>
+
+```php
+use stdClass;
+
+// Using stdClass via import.
+$foo = new stdClass();
+```
+
+```php
+// Using stdClass with its fully qualified name.
+$foo = new \stdClass();
+```
+
+</ValidExample>
+
+### Dynamic properties
+
+`stdClass` objects can be created with dynamic properties by casting an associative array to object or by instantiating a new `stdClass` object and assigning properties to it.
+
+<ValidExample>
+
+```php
+// Object casting.
+$foo = (object) [
+    'bar' => 1,
+    'baz' => 2,
+];
+
+// Setting dynamic properties.
+$foo = new stdClass();
+$foo->bar = 1;
+$foo->baz = 2;
+```
+
+</ValidExample>
+
+### Empty objects
+
+Generic PHP objects can be instantiated by the `new` operator or created by typecasting an empty array to object. Both can be used in a number of situations.
+
+<ValidExample>
+
+Empty object creation
+
+```php
+// Via the `new` operator.
+$foo = new \stdClass();
```

Comment:
```suggestion
$foo = new stdClass();
```
I'd remove the backslash there, it's the unique in the examples and we have already explained above that both are ok (using `use`...).

Reply:
Done! Thanks!